rust/stackable-cockpitd/src/handlers/stacklets.rs

```diff
@@ -13,5 +13,11 @@ pub fn router() -> Router {
     (status = 200, body = Vec<Stacklet>),
 ))]
 pub async fn get_stacklets() -> Json<Vec<Stacklet>> {
-    Json(platform::stacklet::list_stacklets(None).await.unwrap())
+    let client = Client::new().await.unwrap();
```

Comment:
Can we please replace the existing unwrap with an expect?

Reply:
d308722